packages/cli/src/__tests__/e2e/infra.spec.ts

```diff
@@ -430,18 +430,17 @@ describe("e2e tests for infra command", () => {
         [
           "infra",
           "up",
-          "--modules=ganache,dev-server"
+          "--modules=ganache dev-server"
         ],
         getTestCaseDir(1),
       );
 
       await waitForPorts([
-        { port: 8546, expected: true },
-        { port: 8545, expected: true }
+        { port: 8546, expected: true }
```

Comment:
Is this correct? The test currently in `origin-dev` awaits both ports:

```ts
      await waitForPorts([
        { port: 8546, expected: true },
        { port: 8545, expected: true }
      ]);
```

Reply:
Thanks for picking up on this, it was actually an error in how we were parsing arrays, I've fixed it and reverted this test-case modification.